README.md

```diff
@@ -102,11 +102,26 @@ The configuration file mostly corresponds to command-line flags, with the follow
   linters are disabled). `--help` displays the list of default linters with the exact names
   you must use.
 
+#### `Format` key
+
+The default `Format` key places the different fields of an `Issue` into a template. this
+corresponds to the `--format` option command-line flag.
+
+Default `Format`:
+```
+Format: "{{.Path}}:{{.Line}}:{{if .Col}}{{.Col}}{{end}}:{{.Severity}}: {{.Message}} ({{.Linter}})"
+```
+
+#### Format Methods
+
+* `{{.Path.Relative}}` - equivalent to `{{.Path}}` which outputs a relative path to the file
+* `{{.Path.Abs}}` - outputs an absolute path to the file
+
 Here is an example configuration file:
 
 ```json
 {
-  "Enable": ["deadcode", "unconvert"]
+  "Enable": ["deadcode", "unconvert"],
```

Comment:
json doesn't allow trailing commas, so this would actually be an error.

I guess this should also stay with the intro paragraph for "Configuration File", and not under the "Format Methods" section.